pkgs/development/python-modules/edlib/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, edlib
+, cython
+, python
+}:
+
+buildPythonPackage {
+  inherit (edlib) pname src meta;
+  version = "1.3.9";
```

Comment:
If using the latest release instead of unstable then this version can be inherited too.

Reply:
The issue is the python and c++ version numbers don't match.